### ISSUE TITLE
There is a bug when calculating the current date in weekview.ts

### ISSUE THREAD

**Issue by TodorBonchev** [twinssbc/Ionic2-Calendar#590]
In the following line:
https://github.com/twinssbc/Ionic2-Calendar/blob/1b0cc9032fee0fa4e8048dface4e1da263c223cd/src/weekview.ts#L1231
Current day difference return NaN because in Date.UTC the third argument is today.getTime instead of today.getDate.
Here is a suggested fix:
```
            currentDayDifference = Math.floor(
                (
                Date.UTC(today.getFullYear(), today.getMonth(), today.getDate()) 
                - 
                Date.UTC(currentViewStartDate.getFullYear(), currentViewStartDate.getMonth(), currentViewStartDate.getDate())
                ) 
                / oneDay);
```

**Comment by twinssbc** [comment]
@TodorBonchev Good catch! I have published a version 0.6.7 to fix it. Feel free to give a test.

**Comment by TodorBonchev** [comment]
Thanks for the fast response.
